pages/blog/_posts/2020-03-30-catch-imsi-from-vowifi-session.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blogpost
+permalink: /blog/vowifi-imsi-leak/
+title: VoWifi leaking IMSI
+date: 2020-03-30
+tags: telco 4g epdg vowifi networking security
+author: <a href="https://www.linkedin.com/in/rskjetlein/">Roger Skjetlein</a>
+---
+
+4g offers more services than the earlier generation such as 3g and 2g. One of
+the service that really have gained traction later years is VoLTE, that is Voice
```

Comment:
`service` -> `services`
`really have` -> `has really`
`VoLTE, that is voice over LTE` -> `VoLTE (Voice over LTE)`
`and later that we will go more in dept regarding security is VoWifi` // I think some words are missing here.
`VoWifi, Voice over Wifi.` -> `VoWifi (Voice over Wifi).`

Reply:
Fixed